### DISCUSSION TITLE
CI: 9 tests failing in Conda 1.10 env

### DISCUSSION THREAD

**Discussion by daniellepintz** [Lightning-AI/pytorch-lightning#9273] [General]
I noticed in recent commits these 9 tests are consistently failing in the Conda 1.10 environment 
<img width="421" alt="Screen Shot 2021-09-01 at 9 42 35 PM" src="https://user-images.githubusercontent.com/38207072/131782988-f9292eee-d5cb-4e07-9a34-47875deef83c.png">
(https://github.com/PyTorchLightning/pytorch-lightning/runs/3483032531)

```
FAILED tests/callbacks/test_quantization.py::test_quantization[True-True-average]
FAILED tests/callbacks/test_quantization.py::test_quantization[True-False-average]
FAILED tests/callbacks/test_quantization.py::test_quantization[False-True-average]
FAILED tests/callbacks/test_quantization.py::test_quantization[False-False-average]
FAILED tests/models/test_amp.py::test_amp_cpus[1-bf16-None] - TypeError: __in...
FAILED tests/models/test_amp.py::test_amp_cpus[1-bf16-ddp_spawn] - TypeError:...
FAILED tests/models/test_amp.py::test_amp_cpus[2-bf16-None] - torch.multiproc...
FAILED tests/models/test_amp.py::test_amp_cpus[2-bf16-ddp_spawn] - torch.mult...
FAILED tests/models/test_amp.py::test_cpu_model_with_amp - AssertionError: Re...
```

Starting this thread to track this issue and try to resolve. Does anyone have any ideas? cc @Borda

**Comment by Borda** [comment]
@daniellepintz making it rather as an issue the just discussion #9280
